frontend/test/metabase/lib/expressions/typechecker.unit.spec.js

```diff
@@ -104,6 +104,12 @@ describe("type-checker", () => {
     function filter(source) {
       return collect(source, "boolean");
     }
+    function validate(source) {
```

Comment:
I feel like relying on stomping on the `validate` on the outer scope could lead to some weird spooky action-at-a-distance

Reply:
Will refactor that, too. Just don't feel like bucketing it here. Expect a minor follow-up clean-up.